tests/helics/pkg_test_code/main98.cpp

```diff
@@ -6,9 +6,11 @@ SPDX-License-Identifier: BSD-3-Clause
 */
 
 #include "helics/cpp98/helics.hpp"
+#include <iostream>
 
 int main (int argc, char *argv[])
 {
     volatile helicscpp::FederateInfo fi;
+	std::cout<<helicsGetVersion()<<std::endl;
```

Comment:
Use spaces instead of tabs, and space expressions appropriately.

```
    std::cout << helicsGetVersion() << std::endl; 
```

Reply:
In the future it might be simpler to use the suggestion mode then I can commit the suggestions directly or as a batch particularly if they are just formatting or grammer/spelling changes.

Reply:
But I probably should run the cmake through cmake-format again since there were quite a few changes.